databricks-sdk-java/src/main/java/com/databricks/sdk/core/DatabricksConfig.java

```diff
@@ -212,6 +213,13 @@ private synchronized DatabricksConfig innerResolve() {
     }
   }
 
+  /** Sort scopes in-place for better de-duplication in the refresh token cache. */
+  private void sortScopes() {
```

Comment:
This is a private function, so according to Java convention. We shouldn't have the comments in the javadoc format.
```suggestion
  // Sort scopes in-place for better de-duplication in the refresh token cache.
  private void sortScopes() {
```